draft-iab-path-signals-collaboration.md

```diff
@@ -301,41 +301,43 @@ an airgapped network.  Most "closed" networks have at least some needs
 and means to access the rest of the Internet, and should not be
 modeled as if they had an impenetrable security barrier.
 
-## Consent of Parties
-
-Consent and trust must determine the distribution of information. The set of
-entities that need to consent is determined by the scope and specificity of
-the information being shared. 
-
-Three distinct types of consent are recommended for collaboration or
-information sharing:
-
-* A corollary of the intentional distribution is that the sender
-  needs to agree to sending the information. Or that the requester
-  for an action needs to agree to make a request; it should not be an
-  implicit decision by the receiver that information was sent or a
-  request was made, just because a packet happened to be formed in a
-  particular way.
-
-* At the same time, the recipient of information or the target of a
-  request should agree to wishing to receive the information. It
-  should not be burdened with extra processing if it does not have
-  willigness or a need to do so. This happens naturally in most
-  protocol designs, but has been a problem for some cases where
-  "slow path" packet processing is required or implied, and the
-  recipient or router did not have willingness for this.
-
-* Internet communications are not made for the applications, they are
-  ultimately made on behalf of users. Information relating to the
-  users is something that both networks and applications should be
-  careful with, and not be shared without the user's consent. This is
-  not always easy, as the interests of the user and (for instance)
-  application developer may not always coincide; some applications
-  may wish to collect more information about the user than the user
-  would like.
-
-  As a result, typically an application's consent is not the same as
-  the user's consent.
+## Control of the Distribution of Information
+
+Trust and mutual agreement between the involved entities must determine
+the distribution of information, in order to give adequate control to 
+each entity over the collaboration or information sharing.
+
+A corollary of the intentional distribution is that the sender
+needs to agree to sending the information. Or that the requester
+for an action needs to agree to make a request; it should not be an
+implicit decision by the receiver that information was sent or a
+request was made, just because a packet happened to be formed in a
+particular way.
```

Comment:
The grammar here seems odd, and I'm not quite sure what it's trying to say? Is this really a corollary?

Reply:
I didn't write this text originally. Is it any better if we just say "consequence" instead or "Following this principle"?

Reply:
```suggestion
The sender
needs to agree to sending the information. Any passing of information or request
for an action needs to be explicit, and use protocol mechanisms that are designed
for the purpose. Merely sending a particular kind of a packet to a destination should not
be interpreted as an implicit agreement.
```

Reply:
In the above suggestion, we don't need to say whether this is a corollary or not. We can just describe the situation from the sender's perspective.

Reply:
Please review.

Reply:
I created a new commit with your suggested wording and merged it.